### PR TITLE
Fix inconsistency in version variable names and content

### PR DESCRIPTION
**Pull request by maelk**
This creates a CAPM3_VERSION variable and sets the CAPI_VERSION
properly depending on CAPM3_VERSION. It also filters the latest
releases based on the CAPM3_VERSION

### CODE REVIEW

**Comment by maelk** [comment]
The v1a4 should fail until we release CAPM3 v1alpha4

**Comment by maelk** [comment]
yes, that's WIP, check your slack

File: vm-setup/roles/v1aX_integration_test/vars/main.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ IMAGE_OS: "{{ lookup('env', 'IMAGE_OS') | default('Centos', true)}}"
 CONTAINER_RUNTIME: "{{ lookup('env', 'CONTAINER_RUNTIME') | default('podman', true)}}"
 DEFAULT_HOSTS_MEMORY: "{{ lookup('env', 'DEFAULT_HOSTS_MEMORY') | default('4096', true)}}"
 CAPI_VERSION: "{{ lookup('env', 'CAPI_VERSION') | default('v1alpha3', true)}}"
```
[the remark this answers is not in the document]
Comment (maelk) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml:53]: it's mostly for backwards compatibility for now, but we might use it later on

**Comment by maelk** [comment]
the extra parameter is to select the minor version from which we fetch the latest release (v1alpha3 or 4 basically)

**Comment by fmuyassarov** [comment]
> the extra parameter is to select the minor version from which we fetch the latest release (v1alpha3 or 4 basically)

Got it.

**Comment by fmuyassarov** [comment]
v1a4 CI failed because of the issue with `CAPM3RELEASE`
```
 release_tag=null
+++ [[ null == \n\u\l\l ]]
+++ return 1
++ export CAPM3RELEASE=
++ CAPM3RELEASE=
```

**Comment by maelk** [comment]
I have the fix already, just waiting for the v1a3 tests, to check if they pass or not, and fix all at once

**Pull request merged**